app/templates/static/layout.static.pug

```diff
@@ -5,11 +5,11 @@ html(lang='en')
     meta(charset='utf-8')
     meta(name='viewport', content='width=1024')
     meta(http-equiv='Accept-CH', content='Viewport-Width')
-    //- title CodeCombat - Learn how to code by playing a game
     title(data-i18n="new_home.title")
-    meta(data-i18n="[content]new_home.meta_description")
-    meta(property='og:title', content='CodeCombat: Learn to Code by Playing a Game')
-    meta(property='og:url', content='http://codecombat.com')
+    meta(name="keywords", data-i18n="[content]new_home.meta_keywords")
+    meta(name="description", data-i18n="[content]new_home.meta_description")
+    meta(property='og:title', data-i18n="[content]new_home.title")
+    meta(property='og:url', data-i18n="[content]new_home.meta_og_url")
```

Comment:
This should be driven from an environment variable such `COCO_MAIN_HOSTNAME` if possible, or driven from the `chinaInfra` param.  Setting it up this way will change the meta links to koudashijie on CodeCombat whenever a user switches to a ZH locale.  This isn't a big deal for users but this will confuse crawlers.

Reply:
this is driven from the `chinaInfra`, and compiled to static file（in public directory）, according to what I understanding, public file in server can be different between China server and America server, so crawlers will not be confused.

Reply:
In my test, only the title changed when switch to en/zh (title is driven on vue component, I think? ), other properties didn't change